scalike-jdbc-scala-fx/src/test/scala/fx/DatabaseSpec.scala

```diff
@@ -0,0 +1,54 @@
+package fx
+
+import java.sql.SQLException
+import scalikejdbc.*
+import scalikejdbc.DB.*
+import java.util.concurrent.atomic.AtomicInteger
+
+class DatabaseSpec extends DatabaseSuite {
+
+  var counter: AtomicInteger = new AtomicInteger(0)
```

Comment:
```suggestion
  val counter: AtomicInteger = new AtomicInteger(0)
```
The AtomicInteger is mutable, so the reference doesn't have to be mutable.